api/tests/routes/backoffice_v3/accounts_test.py

```diff
@@ -444,6 +445,15 @@ def test_get_public_account(self, authenticated_client, index, expected_badge, e
         if user.dateOfBirth:
             assert f"Date de naissance {user.dateOfBirth.strftime('%d/%m/%Y')}" in content
         assert "Date de naissance déclarée à l'inscription" not in content
+        if user.deposit:
+            assert (
+                f"Crédité le : {user.deposit.dateCreated.astimezone(tz=pytz.timezone('Europe/Paris')).strftime('%d/%m/%Y')}"
+                in content
+            )
+            assert (
+                f"Date d'expiration du crédit : {user.deposit.expirationDate.astimezone(tz=pytz.timezone('Europe/Paris')).strftime('%d/%m/%Y à %Hh%M')} UTC"
```

Comment:
Du coup si tu prends ma suggestion il reste ceci :

```suggestion
                f"Date d'expiration du crédit : {user.deposit.expirationDate.astimezone(tz=pytz.timezone('Europe/Paris')).strftime('%d/%m/%Y à %Hh%M')}"
```